go/interpreter/lfvm/instructions_test.go

```diff
@@ -870,3 +873,189 @@ func TestExpansionCostOverflow(t *testing.T) {
 		}
 	}
 }
+
+func TestSStoreGasCost(t *testing.T) {
+
+	tests := map[string]struct {
+		runtimeSetup   func(*tosca.MockRunContext)
+		revision       tosca.Revision
+		gas            tosca.Gas
+		expectedStatus Status
+	}{
+		"berlin-enough-gas": {
+			runtimeSetup: func(runContext *tosca.MockRunContext) {
+				runContext.EXPECT().IsSlotInAccessList(gomock.Any(), gomock.Any()).Return(true, true).AnyTimes()
```

Comment:
In all these testcases the `AnyTimes()` can be removed.
The `minimum-gas-invariant` case does not need to prime the mock as it does not call `IsSlotInAccessList`.